server/controllers/api/videos/upload.ts

```diff
@@ -210,7 +216,12 @@ async function addVideo (options: {
 
   createTorrentFederate(video, videoFile)
     .then(() => {
-      if (video.state !== VideoState.TO_TRANSCODE) return
+      if (video.state !== VideoState.TO_TRANSCODE) {
```

Comment:
We could create a `MOVE_TO_EXTERNAL_STORAGE` state and display the appropriate message to end users

Reply:
We could also collapse the two things to "processing" if object storage is enabled. The distinction doesn't matter for watching the video, if it's not transcoded or moved you're not getting the optimal experience. In code it already means "processing" so I can rename the state in the enum

Reply:
Yes you're right but I think we should keep 2 different states so it's easier to debug things for admins if there have issues with a specific video